sql/moz-fx-data-shared-prod/ads/fakespot_daily_events_rollup/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Fakespot daily events rollup
+description: |-
+  Daily events rollup from Fakespot, pulled from syndicate dataset
+owners:
+  - cmorales@mozilla.com
+authorized: true
```

Comment:
Actually, it might get auto-skipped by CI since it's authorized but the `authorized` metadata is actually a label e.g. https://github.com/mozilla/bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/accounts_db/fxa_oauth_clients/metadata.yaml#L10

Reply:
Oh, right, thanks for catching that!